automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/OrchestrationApiSpec.scala

```diff
@@ -81,10 +81,6 @@ class OrchestrationApiSpec extends FreeSpec with Matchers with ScalaFutures with
 
         Orchestration.NIH.addUserInNIH(OrchConfig.Users.genericJsonWebTokenKey)
 
-        verifyDatasetPermissions(Set(NihDatasetPermission("TCGA", true), NihDatasetPermission("TARGET", true)))
-
-        Orchestration.NIH.syncWhitelistFull
-
```

Comment:
@MatthewBemis remember adding this test 3 years ago? I don't know if you wrote it then or just moved it from somewhere else but this part seems wrong and exactly what this PR is fixing. When a user links a different NIH account they should lose access to anything the original has access to but the new does not.

Reply:
I think I wrote it but I don't remember the specifics. If I had to guess, this test was backfill for functionality that had already existed in prod for a while so I probably made the dangerous assumption that it was all correct, and then just shaped the test around that incorrect existing behavior. Or it's bad copy-paste from another test case. Either way, it's definitely wrong

Reply:
👍